spec/features/restaurant_spec.rb

```diff
@@ -166,13 +166,13 @@
     sign_in restaurant.user
     visit restaurant_path(restaurant)
 
-    expect(page).to have_content("Delete restaurant")
+    page.should have_selector(:link_or_button, 'Delete restaurant')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.